chain/client/src/stateless_validation/chunk_endorsement_tracker.rs

```diff
@@ -177,27 +191,15 @@ impl ChunkEndorsementTracker {
         let Some(chunk_endorsements) = self.chunk_endorsements.get(&chunk_header.chunk_hash())
         else {
             // Early return if no chunk_enforsements found in our cache.
```

Comment:
```suggestion
            // Early return if no chunk_endorsements found in our cache.
```
As we are here anyway...